cves/CVE-2011-4136.yml

```diff
@@ -65,11 +76,13 @@ fixes_vcc_instructions: |
   Please put the commit hash in "commit" below (see my example in
   CVE-2011-3092.yml). Fixes and VCCs follow the same format.
 fixes:
-- commit: ac7c3a110f906e4dfed3a17451bf7fd9fcb81296
+- commit: 33076af6f2aa5285b3a70246e14163b901b512f7
   note:
-- commit: fbe2eead2fa9d808658ca582241bcacb02618840
-  note:
-vccs: []
+vccs: 
+- commit: bcf7e9a9fe037eff4d5dea0cdd8c35104590e1a8
+  note: This is when the ability to store session data in cache was created.
```

Comment:
May read better if you say something like "This is when caching session data was created"?